README.md

```diff
@@ -216,29 +219,30 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
     - 'names_dmp': file with sars-cov-2 taxonomy names, originally generated for Kaiju
     - 'prot_names.dmp': sars-cov-2 dmp name file used for associating protein with tax ID (tested with DIAMOND). Subset from NCBI taxdmp names.dmp.
     - 'prot_nodes.dmp': sars-cov-2 dmp node file used for associated protein with tax ID (tested with DIAMOND). Subset from NCBI taxdmp nodes.dmp.
-    - 'prot.accession2taxid.gz': sars-cov-2 ORF1ab polyprotein accession ID to tax id file, to match sars-cov-2 proteome.fasta 
+    - 'prot.accession2taxid.gz': sars-cov-2 ORF1ab polyprotein accession ID to tax id file, to match sars-cov-2 proteome.fasta
 
 - homo_sapiens
+
   - 10xgenomics
     - cellranger
-      - test_10x_10k_pbmc_5fb_fastq_\{1,2\}_gz: 5' V2 Feature Barcode FASTQs from 10k PBMC data
-      - test_10x_10k_pbmc_5gex_fastq_\{1,2\}_gz: 5' V2 gene expression FASTQs from 10k PBMC data
-      - test_10x_10k_pbmc_b_fastq_\{1,2\}_gz: 5' V2 B-cell FASTQs from 10k PBMC data
-      - test_10x_10k_pbmc_t_fastq_\{1,2\}_gz: 5' V2 T-cell FASTQs from 10K PBMC data
+      - test*10x_10k_pbmc_5fb_fastq*\{1,2\}\_gz: 5' V2 Feature Barcode FASTQs from 10k PBMC data
+      - test*10x_10k_pbmc_5gex_fastq*\{1,2\}\_gz: 5' V2 gene expression FASTQs from 10k PBMC data
+      - test*10x_10k_pbmc_b_fastq*\{1,2\}\_gz: 5' V2 B-cell FASTQs from 10k PBMC data
+      - test*10x_10k_pbmc_t_fastq*\{1,2\}\_gz: 5' V2 T-cell FASTQs from 10K PBMC data
       - test_10x_10k_pbmc_feature_ref_csv: Feature Barcode reference for the 10k PBMC
-      - test_10x_10k_pbmc_cmo_cmo_fastq_\{1,2\}_gz: 3' V3 Cell Multiplexing FASTQs from 10k PBMC data with Cell Multiplexing
-      - test_10x_10k_pbmc_cmo_gex\{1,2\}_fastq_\{1,2\}_gz: 3' V3 gene expression FASTQs from 10k PBMC data with Cell Multiplexing
+      - test*10x_10k_pbmc_cmo_cmo_fastq*\{1,2\}\_gz: 3' V3 Cell Multiplexing FASTQs from 10k PBMC data with Cell Multiplexing
+      - test*10x_10k_pbmc_cmo_gex\{1,2\}\_fastq*\{1,2\}\_gz: 3' V3 gene expression FASTQs from 10k PBMC data with Cell Multiplexing
       - test_10x_10k_pbmc_cmo_feature_ref_csv: Feature Barcode reference for the 10k PBMC data with Cell Multiplexing
-      - test_10x_5k_cmvpos_tcells_ab_fastq_\{1,2\}_gz: Antibody Capture FASTQs from the 5k CMV+ T-cell dataset
-      - test_10x_5k_cmvpos_tcells_gex1_fastq_\{1,2\}_gz: Gene expression FASTQs from the 5k CMV+ T-cell dataset
-      - test_10x_5k_cmvpos_tcells_vdj_fastq_\{1,2\}_gz: V(D)J FASTQs from the 5k CMV+ T-cell dataset
+      - test*10x_5k_cmvpos_tcells_ab_fastq*\{1,2\}\_gz: Antibody Capture FASTQs from the 5k CMV+ T-cell dataset
+      - test*10x_5k_cmvpos_tcells_gex1_fastq*\{1,2\}\_gz: Gene expression FASTQs from the 5k CMV+ T-cell dataset
+      - test*10x_5k_cmvpos_tcells_vdj_fastq*\{1,2\}\_gz: V(D)J FASTQs from the 5k CMV+ T-cell dataset
```

Comment:
Why were underscores changed to `*` here? That seems odd and unrelated.

Reply:
Oh, I get it. It's the markdown formatter that considered this emphasized text and changed from `_` to `*` instead. I recommend either checking out a previous version of the README or wrapping these strings within backticks ` to avoid this problem.